Gemfile

```diff
@@ -19,11 +21,17 @@ group :deployment, :development do
 end
 
 group :development, :test do
+  if rails_version == "main"
+    gem "rails", github: "rails/rails", branch: "main"
+  else
+    gem "rails", "~> #{rails_version}.0"
```

Comment:
Why do we anchor it on `.0`? Can't we just pass the whole version through `RAILS_VERSION`?